reo/process_results.py

```diff
@@ -320,6 +320,77 @@ def upfront_capex_after_incentives(self):
                                               (1 - self.inputs["Financial"]["owner_tax_pct"])
             return round(upfront_capex_after_incentives, 2)
 
+        @property
+        def get_offgrid_lcoe_breakdown(self):
+
+            lcoe_component_fuel = None
+            lcoe_component_re_capex = None
+            lcoe_component_diesel_capex = None
+            lcoe_component_other_capex = None
+            lcoe_component_om = None
+            lcoe_component_other_annual_costs = None
+            lcoe = self.nested_outputs["Scenario"]["Site"]["Financial"]["microgrid_lcoe_us_dollars_per_kwh"]
+            
+            if lcoe is not None:
+                lcc = self.nested_outputs["Scenario"]["Site"]["Financial"]["lcc_us_dollars"]
+                
+                fuel = (self.nested_outputs["Scenario"]['Site']['Generator']['total_fuel_cost_us_dollars'] or 0)
+                other_capex = (self.nested_outputs["Scenario"]["Site"]["Financial"]["additional_cap_costs_us_dollars"] or 0)
+                other_annual_costs = (self.nested_outputs["Scenario"]["Site"]["Financial"]["total_annual_cost_us_dollars"] or 0)
+                
+                analysis_period = self.inputs["Financial"]["analysis_years"]
+                discount_rate = self.inputs["Financial"]["owner_discount_pct"] # This is set to offtaker_disc_pct if third_party is false
+                # diesel_unit_cost = setup_capital_cost_offgrid(analysis_period, discount_rate,
```

Comment:
@adfarth can we remove all of the commented code in this new method? If not can you please add comments explaining what the commented code is for?

Reply:
@NLaws  I added a comment mentioning that we may want to pull this capability back in in the future. Is that okay?

Reply:
I try to abide by the [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle, but since the code is in the commit history you could just put a note of what the capability is and the hash of the last commit that includes the code. It's fine if you want to leave it too.

Reply:
Ok! I removed the commented code both in process_results.py and data_manager.py